corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -354,8 +354,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         initialize: function (options) {
             const self = this;
             self.styles = options.styles;
-            self.hasNoItems = options.collection.length === 0;
+            self.hasNoItems = options.collection.length === 0 || options.triggerEmptyCaseList;
+            self.headers = options.triggerEmptyCaseList ? [] : this.options.headers;
             self.redoLast = options.redoLast;
+            self.actions = options.sidebarEnabled ? [] : this.options.actions;
```

Comment:
I'm guessing this is so that the main area doesn't show the case search buttons? What happens if registration from the case list is enabled?

Reply:
ah yes this needs to be fixed-- I'll look into isolating just the "Search Again" workflow

Reply:
undoing this commit https://github.com/dimagi/commcare-hq/pull/33060#issuecomment-1579181548:~:text=%E2%80%A6-,fbd1b00,-Merge%20state